lib/rails_drivers/extensions.rb

```diff
@@ -17,7 +17,8 @@ module Extensions
       )
 
       @@driver_extensions = possible_extensions.map do |path|
-        require_dependency path
+        load path
+
```

Comment:
@Resonious Thanks for writing a unit test. I'll take a crack at this again, pretty sure there is some way to achieve this since Rails concerns behave in a similar fashion and are auto-loadable with the Rails framework.